src/assets/scripts/client/aircraft/Pilot/Pilot.js

```diff
@@ -133,18 +134,27 @@ export default class Pilot {
         const altitudeInstruction = radio_trend('altitude', currentAltitude, altitude);
         const altitudeVerbal = radio_altitude(readbackAltitude);
         let expediteReadback = '';
-
+        
         if (expedite) {
             // including space here so when expedite is false there isnt an extra space after altitude
             expediteReadback = ' and expedite';
 
             this.shouldExpediteAltitudeChange();
         }
-
+        
         const readback = {};
         readback.log = `${altitudeInstruction} ${readbackAltitude}${expediteReadback}`;
         readback.say = `${altitudeInstruction} ${altitudeVerbal}${expediteReadback}`;
 
+        if (!aircraftModel.model.isValidAltitude(altitude)) {
+            let altitudeUnattainableReadback = 'requested altitude unattainable, ';
+
+            readback.log = `${altitudeUnattainableReadback}${altitudeInstruction} ${readbackAltitude}${expediteReadback}`;
+            readback.say = `${altitudeUnattainableReadback}${altitudeInstruction} ${altitudeVerbal}${expediteReadback}`;
```

Comment:
Let's simplify all this to like `unable to maintain ${readbackAltitude} due to performance`.